ui/lib/apps/InstanceProfiling/pages/Detail.js

```diff
@@ -134,7 +134,6 @@ export default function Page() {
         loading={isLoading && !data}
         columns={columns}
         items={data?.tasks_status || []}
-        getKey={(row) => row.id}
```

Comment:
`row => row?.id` works as well, but I prefer to just remove it, there is no warning after removing it in the console, even there are multiple rows.